src/category/monad/cont.lean

```diff
@@ -0,0 +1,81 @@
+
+/-
+Copyright (c) 2019 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Simon Hudon
+
+Monad encapsulating continuation passing programming style, similar to
+Haskell's `Cont`, `ContT` and `MonadCont`:
+http://hackage.haskell.org/package/mtl-2.2.2/docs/Control-Monad-Cont.html
+-/
+
+import tactic.interactive
+
+universes u v w
+
+structure label (α : Type w) (m : Type u → Type v) (β : Type u) :=
```

Comment:
`label` or `goto` shouldn't be defined on the top level.

Reply:
You mean that you'd put them in `cont_t`?

Reply:
Hm, maybe. I don't know a good namespace, but having `label` on the top level is a no go for me.